libraries/AP_Mount/AP_Mount_Params.cpp

```diff
@@ -152,6 +152,12 @@ const AP_Param::GroupInfo AP_Mount_Params::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_LEAD_PTCH", 13, AP_Mount_Params, pitch_stb_lead, 0.0f),
 
+    // @Param: _SYSID_DFLT
+    // @DisplayName: Mount Target sysID
+    // @Description: Default Target sysID for the mount to point to
+    // @RebootRequired: True
+    // @User: Standard
+    AP_GROUPINFO("_SYSID_DFLT", 14, AP_Mount_Params, sysid_default, 1),
```

Comment:
I think it makes more sense for the default to be "0" which we could interpret as the first other sysid the vehicle sees.  "1" is the default system id of all vehicles meaning it might try to follow itself.